packages/stark-core/src/logging/logging.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from "@angular/core";
+// import {ActionReducerMap, StoreModule} from "@ngrx/store";
+
+// import {loggingReducer} from "./reducers/index";
+// import {StarkLoggingActions} from "./actions/index";
+import { StarkLoggingServiceImpl, starkLoggingServiceName } from "./services/index";
+import { starkAppConfigConstantName, StarkApplicationConfigImpl } from "../configuration/entities/index";
+// import {StarkLoggingApplicationState} from "../common/store/starkCoreApplicationState";
+
+// const reducers:ActionReducerMap<StarkLoggingApplicationState, StarkLoggingActions> = {
+// 	starkLogging: loggingReducer
+// };
+
+@NgModule({
+	// imports: [StoreModule.forRoot(reducers)],
+	imports: [],
+	declarations: [],
+	providers: [
```

Comment:
Here you can provide a mock for the XSRF service in the meantime (see [HttpModule](https://github.com/NationalBankBelgium/stark/blob/master/packages/stark-core/src/http/http.module.ts)) so at least the service implementation is complete